bitnami/thanos/values.yaml

```diff
@@ -1781,6 +1781,22 @@ ruler:
   ##
   alertmanagers: []
 
+  ## Alertmanagers Configuration
```

Comment:
Let me know if this new setup makes sense! I dropped the `.file` here so that anything under `alertmanagersConfig` is now Thanos configuration. This will also avoid any trouble if Thanos chooses to add a top level `file` argument to their config.